modules/core_project_factory/main.tf

```diff
@@ -443,7 +440,7 @@ resource "google_storage_bucket_iam_member" "api_s_account_storage_admin_on_proj
 resource "google_compute_subnetwork_iam_member" "gke_shared_vpc_subnets" {
   provider = google-beta
 
-  count = local.gke_shared_vpc_enabled && length(compact(var.shared_vpc_subnets)) != 0 ? length(var.shared_vpc_subnets) : 0
+  count = var.shared_vpc_enabled && length(compact(var.shared_vpc_subnets)) != 0 ? length(var.shared_vpc_subnets) : 0
```

Comment:
This logic should not be changed. The definition of `local.gke_shared_vpc_enabled` must be updated to use `var.shared_vpc_enabled`.

Reply:
Fixed